spec/features/community_admin/create_a_new_judge_spec.rb

```diff
@@ -10,15 +10,20 @@
   end
 
   scenario 'creating a new judge' do
-    judge_count = region.judges.count
     visit new_community_admin_judge_path(community)
 
     fill_in 'First Name', with: Faker::Name.first_name
     fill_in 'Last Name', with: Faker::Name.last_name
-    click_button 'Save'
 
-    expect(region.judges.count).to eq (judge_count + 1)
+    expect { click_button 'Save' }.to change { region.judges.count }.from(0).to(1)
```

Comment:
Made this test more succinct with the `expect {}.to change` syntax

Reply:
This is such a great refactor!  Thank you!